llvm/test/CodeGen/SPARC/salvage-debug-isel.ll

```diff
@@ -0,0 +1,69 @@
+; RUN: llc -march=sparc -O1 %s -o - | FileCheck %s
```

Comment:
I think we should use `llc -march=sparc %s -o - -stop-after=finalize-isel` here to reduce the surface area we're testing to the interesting bit (isel) and change the CHECK(s) to:

`CHECK: DBG_VALUE %stack.0.b, $noreg, ![[#]], !DIExpression(DW_OP_plus_uconst, 3, DW_OP_stack_value)`